crates/provider-sdk/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "provider-sdk"
+version = "0.1.0"
+edition = "2021"
```

Comment:
This should be inherited along with other metadata, just like in all other crates in the workspace